### PR TITLE
Added ISIS TLP app

### PR DESCRIPTION
**Pull request by jessemapel**

### CODE REVIEW

File: Applications/ISIS.md
Comment (jlaura) [inline Applications/ISIS.md:11]: I mean these as questions on process. What kind of metrics might we ask for here? What reason might we have for asking for this type of information? (I just read @victoronline PR and saw the blurb about limited mentor resources, so that is the impetus behind these comments.)

File: Applications/ISIS.md
Comment (jlaura) [inline Applications/ISIS.md:25]: Does it make sense to link directly back to the doc where these area articulated on the ISIS-TC side? One of the things I like about condo-forge maintainers is that the person being added as a maintainer needs to +1 the PR. Should the ISIS TC +1 this PR? Or is a link back to this language sufficient?

File: Applications/ISIS.md
Comment (jlaura) [inline Applications/ISIS.md]: Jointly governed by what else? ASC?

File: Applications/ISIS.md
Comment (rbeyer) [reply Applications/ISIS.md:11]: I think any kind the submitting project thinks are relevant.  Usage metrics and/or contribution metrics, I guess.

File: Applications/ISIS.md
Comment (rbeyer) [reply Applications/ISIS.md:25]: I think that if a proto-TC already has a such guidelines in place, then a link to their document should be sufficient (preferred, in fact over re-iterating it).

The "ISIS TC" has already submitted the PR, do they need to also +1 it?  Seems redundant.

File: Applications/ISIS.md
Comment (rbeyer) [reply Applications/ISIS.md]: Yeah, that's a little weird, maybe relict language from when the ASC was in that sentence?

**Comment by victoronline** [comment]
Are we OK with Merging this PR? We haven't really discussed in a bit except that it needs to go into 'Incubation' ASAP and an Issue needs to be created. Just checking before it gets merged. I was also trying to get changes completed to the Lifecycle document to make sure we follow for this application.

File: Applications/ISIS.md
Comment (rbeyer) [reply Applications/ISIS.md]: I think this sentence still needs some polishing.

**Comment by rbeyer** [comment]
I think @jlaura needs to revisit some of these things that he's raised, and either resolve the convo or refine his requests for change.

**Comment by jlaura** [comment]
One open comment still needs to be resolved. Then I believe this is good to merge.

**Comment by jessemapel** [comment]
@jlaura @rbeyer I fixed the governance line. I thought about adding more details, but I don't think they are needed.

**Review by rbeyer — APPROVED** [review]

**Review by AndrewAnnex — APPROVED** [review]

**Comment by rbeyer** [comment]
@victoronline okay, so I think this is good to merge, but since you're the "mentor" you should do it and exercise the workflow.

**Comment by victoronline** [comment]
Application is accepted and project will now enter incubation. Merging Pull Request.

**Comment by victoronline** [comment]
Incubation Issue for ISIS TLP #106

**Comment by victoronline** [comment]
The Planetary Software Organization has accepted your application!

Your ISIS TLP is now under Incubation with
the Planetary Software Organization.  We'll use Issue #106
to track your status through Incubation.  Please make sure 
you are subscribed to that Issue, so you can respond to items
that come up there.

**Pull request merged**
